2.4/examples/replica/mongodb-clustered.json

```diff
@@ -94,7 +94,13 @@
                 "command": [
                   "run-mongod", "initiate"
                 ],
-                "containerName": "mongodb"
+                "containerName": "mongodb",
+                "env": [
+                  {
+                    "name": "MONGODB_REPLICA_NODES",
+                    "value": "3"
```

Comment:
@bparees I've renamed it to `MONGODB_NODES_TO_WAIT`. If you can suggest better name I would be glad.